### DISCUSSION TITLE
[Question]: Disable inbuilt AI providers

### DISCUSSION THREAD

**Discussion by ExploracuriousAlex** [danny-avila/LibreChat#4360] [Q&A] [answered]
### What is your question?

Is there a configuration option to disable some of the inbuilt AI providers that I don't want to use anyway?
Disabling in a way that they don't show up in the drop down.

### More Details

n.a.

### What is the main subject of your question?

Other

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment, marked as answer]
Yea, configure the ENDPOINTS environment variable 
: https://www.librechat.ai/docs/configuration/dotenv#general-config

**Reply by ExploracuriousAlex** [reply]
Great, thanks! I was looking into librechat.yaml all time and oversaw this chapter.
